modules/crm_tools/crm_tools.libraries.yml

```diff
@@ -0,0 +1,21 @@
+open-iconic:
+  remote: https://github.com/iconic/open-iconic
+  version: "1.1.1"
+  license:
+    name: MIT
+    url: https://github.com/iconic/open-iconic/blob/1.1.1/ICON-LICENSE
+    gpl-compatible: true
+  js:
+    js/open-iconic.drupal.js: {}
```

Comment:
This is actually an admin control for icons, so should be a separate library, as this library will need including any time we use open iconic.